src/ray/protobuf/runtime_env_common.proto

```diff
@@ -66,16 +66,84 @@ message PluginRuntimeEnv {
   repeated Plugin plugins = 1;
 }
 
-/// The jars type runtime env.
-message JarsRuntimeEnv {
-  /// TODO(SongGuyang)
+message PythonRuntimeEnv {
+  /// List of URIs that contain the dependent python modules.
+  /// Valid URIs include:
+  /// - A local directory: it will be added to to the PYTHONPATH;
+  ///   - Example: `file:///path/to/my_module`.
+  /// - A zip file in an external storage: it will be unpacked and added to the
+  /// PYTHONPATH.
+  ///   - Example: `http://example.com/my_module.zip`.
+  repeated string dependent_modules = 1;
```

Comment:
why are we renaming this from the python API? this seems confusing to me. let's keep it consistent please.

Reply:
I renamed this field because I have moved it from `RuntimeEnv` protobuf to `PythonRuntimeEnv` protobuf. So, I think the prefix of `py_` is not needed. But I didn't rename the user level API in current PR. Maybe we can keep the user level API consistent. Do you think we should also use the name `py_module` in protobuf here?

Reply:
I would prefer to keep things as consistent with the API as possible, otherwise it makes reading the code much more confusing

Reply:
Make sense. Revert the name.